### ISSUE TITLE
KeyError: 'currentPrice' for crypto currencies

### ISSUE THREAD

**Issue by vutsalsinghal** [pbrissaud/suivi-bourse#92]
**Describe the bug**
Hi There

I'm getting python KeyError: 'currentPrice' in the logs. I'm using the latest arm64 image ([this](https://hub.docker.com/layers/paulbrissaud/suivi-bourse-app/latest/images/sha256-7b0cedbec06846526a7b4e03ad20a2842920074197200ce359ffc6b37f437de1?context=explore)). I see this error only for crypto currencies (tried a few like BTC-USD, BTC-EUR, ETH-USD). Upon further digging, looks like the issue is in the yfinance library.

```
import yfinance as yf

ticker_info = yf.Ticker("BTC-USD").info
last_quote = ticker_info['currentPrice']
print(last_quote)

Traceback (most recent call last):
  File "main.py", line 13, in <module>
    last_quote = ticker_info['currentPrice']
KeyError: 'currentPrice'
```
There is no key called "currentPrice" for cryptos. Will have to handle that

**Installation mode:** (choose one)
- [ ] Standalone
- [ ] Docker 
- [x] Docker-Compose

**Additional comments**
I had worked on something similar (python finance/portfolio dashboard) and I cant recall but there were several inconsistency issues with the data ingestion from yfinance but things might've changed for better since I last used. In that project, after typing yfinance, I had decided to use [yahoo_fin](http://theautomatic.net/yahoo_fin-documentation/) instead. Worked fine for cypto/non-crypto. 

I'm in no way/shape/form have any association with yahoo_fin. Cheers! 😄

**Comment by pbrissaud** [comment]
Hi,
Thanks for submitting this issue. I will try to find an other way to get the current price and I will open a PR associated to this .

**Comment by pbrissaud** [comment]
@vutsalsinghal PR is merged and I created a `3.3.0` release with the fixes
